examples/SLAM/README.md

```diff
@@ -0,0 +1,20 @@
+# SLAM (World tracking) camera demo
+![Preview](previews/SLAM-example.webp?raw=true "Dynamically generated mesh")
```

Comment:
Nice to have the preview 👍 Could also be referenced in the "Examples" section in the root readme.